src/sortStudents.ts

```diff
@@ -1,16 +1,58 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
```

Comment:
I'm not sure exactly, but I think it should be like this
```suggestion
  Name,
  Surname,
  Age,
  Married,
  AverageGrade,
}

Reply:
no, its ok